homeassistant/components/overkiz/climate_entities/somfy_thermostat.py

```diff
@@ -0,0 +1,191 @@
+"""Support for TaHoma Smart Thermostat."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizState
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_HEAT,
+    CURRENT_HVAC_IDLE,
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT,
+    PRESET_AWAY,
+    PRESET_HOME,
+    PRESET_NONE,
+    SUPPORT_PRESET_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+COMMAND_EXIT_DEROGATION = "exitDerogation"
+COMMAND_REFRESH_STATE = "refreshState"
+COMMAND_SET_DEROGATION = "setDerogation"
+COMMAND_SET_MODE_TEMPERATURE = "setModeTemperature"
+
+CORE_DEROGATED_TARGET_TEMPERATURE_STATE = "core:DerogatedTargetTemperatureState"
+CORE_DEROGATION_ACTIVATION_STATE = "core:DerogationActivationState"
+
+PRESET_FREEZE = "Freeze"
+PRESET_NIGHT = "Night"
```

Comment:
```suggestion
PRESET_FREEZE = "freeze"
PRESET_NIGHT = "night"
```

I think best would be to standardize all presets to use lowercase. Apparently, we cannot translate them (yet). However, I hope to add this soon to core. 
If we want uppercase, we need to do that in AtlanticElectricalHeater as well.

Reply:
Done